statsmodels/tsa/tests/test_arima.py

```diff
@@ -2210,6 +2210,13 @@ def test_ARIMA_exog_predict():
     assert_allclose(dpredict_002, res_d002[-len(dpredict_002):], rtol=1e-4, atol=1e-6)
 
 
+def test_arima_fit_mutliple_calls():
+    y = [-1214.360173, -1848.209905, -2100.918158, -3647.483678, -4711.186773]
+    mod = ARIMA(y, (1, 0, 2))
+    # Make multiple calls to fit
+    mod.fit(disp=0, start_params=[np.mean(y), .1, .1, .1])
+    mod.fit(disp=0, start_params=[np.mean(y), .1, .1, .1])
+    assert_equal(mod.exog_names,  ['const', 'ar.L1.y', 'ma.L1.y', 'ma.L2.y']) 
```

Comment:
to make the test stronger, you can copy the same assert_equal also after the first call to `fit`